### PR TITLE
test: [PROD-12177] update e2e tests to check enable/disable delete ro…

### PR DESCRIPTION
**Pull request by nborde-CSM**
…ws button

Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+import { BreweryParameters, Login } from '../../commons/actions/brewery';
+import { ScenarioParameters, ScenarioManager } from '../../commons/actions/generic';
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+import { Scenarios } from '../../commons/actions';
+import { ScenarioSelector } from '../../commons/actions/generic/ScenarioSelector';
+import utils from '../../commons/TestUtils';
+
+const SCENARIO_DATASET = DATASET.BREWERY_ADT;
+const SCENARIO_RUN_TEMPLATE = RUN_TEMPLATE.BASIC_TYPES;
```
Comment (esasova) [inline cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js:12]: why do we need to assign the variable to another variable used only once ? :thinking:

File: cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+function forgeScenarioName() {
+  const prefix = 'Scenario to rename - ';
```
Comment (esasova) [inline cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js:15]: Can we change the prefix, it must come from other test :slightly_smiling_face:

Comment (nborde-CSM) [reply cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js:15]: Copy/past mess...

File: cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { BreweryParameters, Login } from '../../commons/actions/brewery';
+import { ScenarioParameters, ScenarioManager } from '../../commons/actions/generic';
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+import { Scenarios } from '../../commons/actions';
+import { ScenarioSelector } from '../../commons/actions/generic/ScenarioSelector';
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js:8]: Copy/past mess...

File: cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js (hunk 1 above)
Comment (nborde-CSM) [reply cypress/e2e/brewery/TableParameters-file-deleteRowButton.cy.js:12]: It is inspired by what has already been made in other tests, by exemple in RenameScenario.
but indeed, it is not an absolute requirement.

**Review by esasova — APPROVED** [review]
LGTM :+1:

**Pull request merged**